.env

```diff
@@ -0,0 +1,7 @@
+ACCOUNT_SID=ACcd05917f7ea7e479a69506039b1d0898
+AUTH_TOKEN=ccf501f3df91ed70d4d426fcf2c6553d
+TEST_ACCOUNT_SID=ACcd05917f7ea7e479a69506039b1d0898
+TEST_AUTH_TOKEN=ccf501f3df91ed70d4d426fcf2c6553d
+PHONE_NUMBER=+447766444001
+FROM_PHONE=+441133206201
+#JJ2VEWD3MPiSYmb652NxmhmmTv88JkgmEcliu7oO
```

Comment:
Yes, good point! I was struggling to run Travis CI as when env file wasn't uploaded it didn't pass rspec tests, raising errors where it needed env data. Is there a way to go around it?